packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -130,10 +130,7 @@ const COLUMNS_VM_PV = [
   },
   {
     itemRenderer: ({ vdi }) => (
-      <Size
-        value={vdi.size || null}
```

Comment:
What was the `|| null` for?

Reply:
https://github.com/vatesfr/xen-orchestra/pull/1618/files#diff-e273ddd9a353509a816fbe21cfadc100

Reply:
I guess it was supposed to be a "don't show anything" value in case `vdi.size` was `undefined`. But `editable/Size` doesn't handle `null`. So I think we could use `defined` here to keep the previous behaviour (even though I don't think `vdi.size` could be `undefined`) and handle the `null` case in `Size`.